express-delete/index.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express');
+const app = express();
+
+const grades = [
+  {
+    id: 1,
+    name: 'Rachel Beacham',
+    course: 'Node.js',
+    grade: 100
+  },
+  {
+    id: 2,
+    name: 'Ruby Schaffer',
+    course: 'JavaScript',
+    grade: 95
+  }
+];
+
+app.get('/api/grades', (req, res) => {
+  res.json(grades);
+});
+
+app.delete('/api/grades/:id', function (req, res) {
+  const id = parseInt(req.params.id);
+  const index = id - 1;
```

Comment:
It's really important not to think of ids and "indexes with math". I'm going to make a revision to the exercise to help with this, but imagine if you had 5 grades and the one with id 1 got deleted, then this approach wouldn't work anymore.